core/src/main/java/jeeves/config/springutil/JeevesDelegatingFilterProxy.java

```diff
@@ -24,39 +24,25 @@
 package jeeves.config.springutil;
 
 import com.google.common.annotations.VisibleForTesting;
-
 import org.fao.geonet.ApplicationContextHolder;
 import org.fao.geonet.domain.User;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.web.context.WebApplicationContext;
 import org.springframework.web.filter.DelegatingFilterProxy;
 import org.springframework.web.filter.GenericFilterBean;
 
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
 import java.io.IOException;
-import java.net.InetAddress;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.UnknownHostException;
-import java.util.Arrays;
 import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
-
-import javax.servlet.Filter;
-import javax.servlet.FilterChain;
-import javax.servlet.ServletContext;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
 
 /**
  * Created with IntelliJ IDEA. User: Jesse Date: 11/13/13 Time: 5:15 PM
  */
 public class JeevesDelegatingFilterProxy extends GenericFilterBean {
-    private final static InheritableThreadLocal<String> applicationContextAttributeKey = new InheritableThreadLocal<String>();
-    String trustedHost;
-    private HashMap<String, Filter> _nodeIdToFilterMap = new HashMap<String, Filter>();
+    private final static InheritableThreadLocal<String> applicationContextAttributeKey = new InheritableThreadLocal<>();
+    private final Map<String, Filter> _nodeIdToFilterMap = new HashMap<>();
```

Comment:
I would remove the `_` in the variable name.

Reply:
Renamed to `nodeIdeToFilterMap`.